content/1_General/Debugging_General.mdx

```diff
@@ -80,6 +81,7 @@ Of course, many of these are also relevant for RE / TLE.
 
  - Do you have any possible infinite loops?
  - What is the complexity of your algorithm?
+   - Look back at the problem constraints. What complexity will run in time for that problem?
```

Comment:
I think this is implied (why would you calculate the complexity without looking at the constraints?)